### ISSUE TITLE
Pending force closing - Limbo

### ISSUE THREAD

**Issue by lollenderrofler** [Ride-The-Lightning/RTL#345]
Hi there,

I'm kind of new to Bitcoin and Lightning and have been using RTL on MyNode for a few months now. Today I discovered I was missing 2 million sats of my total balance. I looked around and found something under "pending force closing". Seems like a peer has closed one of my channels, this is the transaction ID:
823d16d41248aabc527912528d3b9a5cea77df4cce29387c1f83579b37c7bfed
The transaction was included in the Blockchain almost two days ago, but I can't see any on-chain-balance. Also RTL shows this as pending with "limbo balance". 

What happens to the balance? Is there some time-lock after which the balance shows up in RTL / LND again? And why did it have to force-close as my own node was online all the time, couldn't it just close with both channel partners cooperatively signing the tx?

**Comment by saubyk** [comment]
Hi, would you happen to know, how much of the channel balance on the local side?
Did you open the channel or your peer? The balance that you will get back on chain is only the local balance.

If its showing a limbo balance, than it might also be showing the block count till maturity.

**Comment by lollenderrofler** [comment]
The channel balance was 4,000,000 sats, a bit more than half of it was on the local side. I did open the channel. Yes, I'm aware that I'll only get back the local balance.

I now found the block count till maturity, thank you! This helped me a lot!
What I still don't get is, as I read the Lightning whitepaper, I thought if both peers are online and one wants to close the channel (cooperatively) on a state the other peer agrees, both can access their on-chain-funds as soon as the tx gets mined. But it seems I was wrong on that. Can someone please clarify why I have to wait for maturity and the peer that closes the channel can access the funds right away?

**Comment by saubyk** [comment]
Look at the 'Close Type' reason in the closed channel history. If the reason is 'Remote Force Close', it's likely that your peer closed the channel when your node might have been offline.

If both the nodes are online, the close type should be 'Cooperative Close' and you should get access to your funds as soon as the transaction is mined.

Also in case of Force close, your peer has to wait for the maturity too, to gain access to the funds.

**Comment by lollenderrofler** [comment]
It's still listed under "pending", as soon as it reaches maturity (about 300 blocks from now) I will take a look at the closed history again.
My node tells me it has been online since over a week now, that was when I triggered the last update. As I have it running on a NAS backed with an extra battery (the battery also powers my router), I can't really imagine that I was offline when the remote peer tried to close the channel. But we'll see in about 300 blocks!
Also, when inspecting the txid (823d16d41248aabc527912528d3b9a5cea77df4cce29387c1f83579b37c7bfed) on a blockexplorer, I can see that the other peer already spent his funds (mined in the same block as the closing transaction), so he did not have to wait for maturity.

Thank you for your help, I think this issue can be closed now as I do know that I just have to wait till maturity. And I'm sorry, during the conversation I realized that this is more a general Lightning / LND troubleshooting issue than it is a RTL one.
